CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ Changelog
   * bridge_domain_vni (@rkorlepa)
 * Encapsulation Profile
   * vni_encapsulation_profile (@rkorlepa)
+* Interface Bridge Domain
+  * interface bdi (@rkorlepa)
```

Comment:
This is not a new provider, it's an extension to an existing provider. Instead of this you can just add an entry at line 27 that says:

`* Added support for bdi interfaces to interface provider`